src/recordlinker/config.py

```diff
@@ -42,5 +48,53 @@ class Settings(pydantic_settings.BaseSettings):
         default="assets/initial_algorithms.json",
     )
 
+    def default_log_config(self) -> dict:
```

Comment:
Is there a purpose to having this default log config, vs. defining the production log config and setting as default?

Reply:
The `production_log_config.json` was created as an external asset to give customers the ability to customize that in their own installation.  If they're not happy with the defaults, they can copy that file, make adjustments and set LOG_CONFIG env variable to point to their version.

We could do this for devs to, but it just seemed like an extra file that we really didn't need to manage.  As a developer its easy to just adjust this method for a short period of time if we want to test something out.  Its easy for developers to temporarily and quickly change code, but not for customers.